opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -249,6 +249,26 @@ message Span {
   // An optional final status for this span. Semantically when Status isn't set, it means
   // span's status code is unset, i.e. assume STATUS_CODE_UNSET (code = 0).
   Status status = 15;
+
+  // ParentSpanIsRemote indicates whether the span's parent is remote (true / false) or
+  // if the field is unset.
+  enum ParentSpanIsRemote {
```

Comment:
Either this should be made generic (OptionalBool with values UNSET, FALSE, TRUE) or it should get more concrete names for the enumerators (PARENT_SPAN_UNKNOWN_REMOTE, PARENT_SPAN_NOT_REMOTE, PARENT_SPAN_REMOTE?). Seeing that the more concrete names are a bit awkward to come up with, I'd slightly prefer the generic OptionalBool, but am fine with either.